docs/monitors/kubernetes_events_monitor.md

```diff
@@ -109,7 +89,5 @@ This monitor was released and enabled by default in Scalyr Agent version `2.0.43
 | `message_log`            | Optional (defaults to ``kubernetes_events.log``). Specifies the file name under which event messages are stored. The file will be placed in the default Scalyr log directory, unless it is an absolute path | 
 | `event_object_filter`    | Optional (defaults to ['CronJob', 'DaemonSet', 'Deployment', 'Job', 'Node', 'Pod', 'ReplicaSet', 'ReplicationController', 'StatefulSet', 'Endpoint']). A list of event object types to filter on. Only events whose ``involvedObject`` ``kind`` is on this list will be included.  To not perform filtering and to send all event kinds, set the environment variable ``SCALYR_K8S_EVENT_OBJECT_FILTER=null``. | 
 | `leader_check_interval`  | Optional (defaults to 60). The number of seconds to wait between checks to see if we are still the leader. | 
-| `leader_node`            | Optional (defaults to None). Force the `leader` to be the scalyr-agent that runs on this node. | 
-| `check_labels`           | Optional (defaults to False). If true, then the monitor will check for any nodes with the label `agent.config.scalyr.com/events_leader_candidate=true` and the node with this label set and that has the oldestcreation time will be the event monitor leader. | 
```

Comment:
Since this is technically a breaking change, let's please also add some info on it to the release notes file.

Reply:
Will do.  Will make a separate PR for this